.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+services:
+  - docker
+
+install:
+  - npm install -g dockerlint
+  - docker pull ubuntu:16.04
```

Comment:
Yikes, i forgot this line. Thanks!
Either way that line is not necessary, it will download the image at the `docker build` command.